Document/SeoAwareInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace ONGR\RouterBundle\Document;
```

Comment:
@einorler please update your IDE settings to leave empty lines before and after license header. Also you should use `/*` instead of `/**` as this is just multiline comment but not DocBlock.